<launch>
  <!-- Urdf -->
   <arg name="model" default="$(find edumip_my_robot)/urdf/edumip_my_robot.urdf"/>
   <arg name="gui" default="true" />
   <arg name="rvizconfig" default="$(find edumip_my_robot)/rviz/urdf.rviz" />

   <param name="robot_description" command="$(find xacro)/xacro.py $(arg model)" />
   <param name="use_gui" value="$(arg gui)"/>

   <node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" />
   <node name="rviz" pkg="rviz" type="rviz" args="-d $(arg rvizconfig)" required="true" />




  <!-- Joy_node -->
   <node pkg="joy" type="joy_node" name="joy_node" output="screen"/>
     <param name="dev" type="string" value="/dev/input/js1" />
     <param name="deadzone" value="0.05" />


  <!-- joy_twist -->
   <node pkg="edumip_my_robot" type="edumip_my_robot_publisher" name="edu_mip_my_robot_publisher_node" output="screen"/>
   <node pkg="joy_twist" type="joy_twist" name="joy_twist_node" output="screen"/>
  <!-- Axes -->
   <param name="axis_linear" value="1" type="int"/>
   <param name="axis_angular" value="0" type="int"/>
   <param name="scale_linear" value="2" type="double"/>
   <param name="scale_angular" value="2" type="double"/>




</launch>
